lxd/storage.go

```diff
@@ -27,6 +27,16 @@ import (
 // Note that any access to this map must be done while holding a lock.
 var lxdStorageOngoingOperationMap = map[string]chan bool{}
 
+type containerCreationMap struct {
+	refcount map[string]int
```

Comment:
That kind of handmade refcouting is a bit weird in Go, why not using sync.WaitGroup as we are doing in a bunch of other places?